scripts/defaults.toml

```diff
@@ -0,0 +1,21 @@
+[control]
+outdir = "."
+save_integrals = false
+integral_format = "fcidump"
+
+[system]
+moltype = "1d"
+bc = "open"
+norb = 0
+nelec = 0
+
```

Comment:
Also, do you think we can/should add default symmetry here? Make it 4 as default

Reply:
symmetry has been included in the defaults.toml file and is set to 4